mozilla-central/setup

```diff
@@ -179,7 +192,55 @@ if [ -n "$INDEXED_GIT_REV" ]; then
             fi
         done
         set -x
+        popd
+
+        date
+
+        # On Windows, analysis for headers ends up in __GENERATED__/dist/include/...
```

Comment:
These comments only talk about Windows, but it sounds like this is done for all platforms? Are there files you know that this helps, or is it only for completeness?

Reply:
It only helps on windows, but I did it for all platforms for completeness.